server/src/test/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepositoryManagerTest.java

```diff
@@ -138,6 +139,6 @@ public void testHas() throws IOException {
 
     private GitRepositoryManager newRepositoryManager() {
         return new GitRepositoryManager(mock(Project.class), rootDir(), ForkJoinPool.commonPool(),
-                                        mock(RepositoryCache.class));
+                                        Executors.newSingleThreadExecutor(), mock(RepositoryCache.class));
```

Comment:
If the `purgeWorker` is used really, it can lead to a dangling thread. How about using `MoreExecutors.directExecutor()`?

Reply:
fix it :-)